app/views/interviews/show.html.slim

```diff
@@ -38,8 +38,9 @@
             |#{q.id}
   - @questions.each do |q|
     .summary-card id="question#{q.id}"
+      - iq = InterviewQuestion.where(interview_id: @interview.id, question_id: q.id).first
```

Comment:
use relations and instance methods for this.
P.S. this is not first time when i remind you to use relations